_topic_map.yml

```diff
@@ -484,9 +484,34 @@ Topics:
     File: api-server
   - Name: Securing service traffic using service serving certificates
     File: service-serving-certificate
+  - Name: Certificate validation
```

Comment:
The certificate validation section looks out of place here. Since the only really useful information there is "API server client certificate expiration is less than five minutes.", I think you can get away with adding the content from Certificate Validation into the User-provided certificates for the API server file instead.

Reply:
ah, thanks! wasn't familiar enough with this stuff to figure out where to stash that :slightly_smiling_face: